### PR TITLE
DOMA-2218 fixed display of tooltips in tickets view

### PR DESCRIPTION
**Pull request by Alllex202**

### CODE REVIEW

**Comment by Alllex202** [comment]
Before

Снимок экрана 2022-07-19 в 15 15 23

After

Снимок экрана 2022-07-19 в 15 16 05

**Review by dkoviazin — APPROVED** [review]

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/common/components/Table/Renders.tsx [hunk 1]
```diff
@@ -70,7 +71,7 @@ type TGetHighlightedFN = (search?: FilterValue | string, postfix?: string | Reac
 export const getHighlightedContents: TGetHighlightedFN = (search, postfix, extraProps, extraPostfixProps = {}, extraTitle) => (text) => {
     // Sometimes we can receive null/undefined as text
     const renderText = text ? String(text) : ''
-    const title = extraTitle ? extraTitle : `${text} ${isString(postfix) && postfix || ''}`
+    const title = extraTitle ? extraTitle : extraTitle === null ? null : `${text} ${isString(postfix) && postfix || ''}`
```
Comment (nomerdvadcatpyat) [inline apps/condo/domains/common/components/Table/Renders.tsx:74]: use `isNull` from lodash

File: apps/condo/domains/common/components/TextHighlighter.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ export type TTextHighlighterProps = {
 
 export const TextHighlighter: React.FC<TTextHighlighterProps> = (props) => {
     const { text, search, renderPart, type, style, title: titleFromProps, children } = props
-    const title = titleFromProps ? titleFromProps : text
+    const title = titleFromProps ? titleFromProps : titleFromProps === null ? null : text
```
Comment (nomerdvadcatpyat) [inline apps/condo/domains/common/components/TextHighlighter.tsx:27]: use `isNull` from lodash

File: apps/condo/domains/common/components/Table/Renders.tsx (hunk 1 above)
Comment (akarjakin) [inline apps/condo/domains/common/components/Table/Renders.tsx:74]: `const title = extraTitle || isNull(extraTitle) ? extraTitle : ...`

File: apps/condo/domains/common/components/TextHighlighter.tsx (hunk 2 above)
Comment (akarjakin) [inline apps/condo/domains/common/components/TextHighlighter.tsx:27]: please don't nest ternary expressions

`const title = titleFromProps || isNull(titleFromProps) : titleFromProps === null : text `

File: apps/condo/domains/common/components/Table/Renders.tsx (hunk 1 above)
Comment (Alllex202) [reply apps/condo/domains/common/components/Table/Renders.tsx]: done

File: apps/condo/domains/common/components/TextHighlighter.tsx (hunk 2 above)
Comment (Alllex202) [reply apps/condo/domains/common/components/TextHighlighter.tsx]: done

File: apps/condo/domains/common/components/Table/Renders.tsx (hunk 1 above)
Comment (Alllex202) [reply apps/condo/domains/common/components/Table/Renders.tsx]: done

File: apps/condo/domains/common/components/TextHighlighter.tsx (hunk 2 above)
Comment (Alllex202) [reply apps/condo/domains/common/components/TextHighlighter.tsx]: done

File: apps/condo/domains/ticket/utils/clientSchema/Renders.tsx [hunk 3]
```diff
@@ -180,7 +180,7 @@ export const getStatusRender = (intl, search: FilterValue) => {
         const { primary: backgroundColor, secondary: color } = status.colors
         const extraProps = { style: { color } }
         // TODO(DOMA-1518) find solution for cases where no status received
-        const highlightedContent = getHighlightedContents(search, null, extraProps)(status.name)
+        const highlightedContent = getHighlightedContents(search, null, extraProps, null,  null)(status.name)
```
Comment (SavelevMatthew) [inline apps/condo/domains/ticket/utils/clientSchema/Renders.tsx:183]: Why explicit nulls?

**Pull request merged**